### PR TITLE
July changes

### PR DESCRIPTION
**Pull request by malinmalliyawadu**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,6 +7,20 @@ import CollectionsSection from "./sections/Collections/Collections";
...
+
+const Footer = styled.footer`
+  padding: 1rem;
+  color: white;
+
+  a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 0.5rem;
```
Comment (maisiembleton) [inline src/App.js:21]: whats gap

File: src/App.js [hunk 2]
```diff
@@ -20,6 +33,17 @@ const App = () => {
...
       <PartnersSection />
+
+      <Footer>
+        <a
+          href="https://preciousplastic.com/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <small>Inspired by Precious Plastic</small>
+          <img src={preciousplastic} alt="Precius Plastic logo" height="30px" />
```
Comment (maisiembleton) [inline src/App.js:44]: Precious - spelt wrong

File: src/sections/Partners/Partners.js [hunk 3]
```diff
@@ -3,85 +3,66 @@ import ImagesBlock from "../../components/ImagesBlock/ImagesBlock";
...
-import hoopla from "../../imgs/partners/hoopla.jpg";
-import littleyellowbird from "../../imgs/partners/littleyellowbird.png";
+import endgame from "../../imgs/partners/EndGame.png";
+import switchedonbikes from "../../imgs/partners/switched-on-bikes.png";
 
 const partnersimgs = [
   {
     img: paperkite,
-    alt: "home-hopper",
+    alt: "PaperKite",
```
Comment (maisiembleton) [inline src/sections/Partners/Partners.js:19]: thanks for fixing this up

**Review by maisiembleton — CHANGES_REQUESTED** [review]
LGTM - with the exception pf a typo

File: src/App.js (hunk 2 above)
Comment (malinmalliyawadu) [reply src/App.js]: 👁👄👁

**Review by maisiembleton — APPROVED** [review]
ye good

**Pull request merged**